modules/private/private.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Module Name: Private site template
+ *
+ * @package Jetpack
+ */
+
+nocache_headers();
+header( 'Content-Type: ' . get_bloginfo( 'html_type' ) . '; charset=' . get_bloginfo( 'charset' ) );
+
+$redirect_to = 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
```

Comment:
Should conditionally set the scheme here. See https://developer.wordpress.org/reference/functions/set_url_scheme/

Reply:
Implemented in [66f1bbc](https://github.com/Automattic/jetpack/pull/10333/commits/66f1bbc7580ade14c8af119143536e4845e14311)